engine/system/core/src/main/java/it/unibz/inf/ontop/materialization/impl/DefaultMaterializedGraphResultSet.java

```diff
@@ -182,6 +183,12 @@ public RDFFact next() throws OntopQueryAnsweringException {
         return null;
     }
 
+    //TODO implement a closable iterator
+    @Override
+    public RDFFactCloseableIterator iterator() {
```

Comment:
Please use the interface `OntopCloseableIterator<RDFFact, OntopConnectionException>` instead